src/include/mpir_request.h

```diff
@@ -367,6 +367,16 @@ MPL_STATIC_INLINE_PREFIX int MPIR_Request_completion_processing_fastpath(MPI_Req
 int MPIR_Request_completion_processing(MPIR_Request *, MPI_Status *, int *);
 int MPIR_Request_get_error(MPIR_Request *);
 
+/* The following routines are helper to handle ULFM.
+ * This needs to be macro because of MPID_Comm_As_enabled. */
```

Comment:
I don't understand the comment here that this needs to be a macro. There should be no problem evaluating the return value of a function call as part of a larger expression.

Reply:
A static inline function would be preferable.

Reply:
`MPID_Comm_As_enabled` is not exposed at the MPIR* level,  but in individually `src/mpid/ch3/include/mpidpre.h` and `src/mpid/ch4/src/ch4_comm.h`.

So I'm not sure what to include in `mpir_request.h` so that the symbol is available.

Reply:
ok resolved using `extern`

Reply:
Hmm, that actually sounds like a ch4 bug. All `MPID_` interfaces are meant to be exposed upwards to MPIR.

Reply:
@raffenet  Any suggestion ? To turn it to function, I have to "extern" 3 symbols actually (you can checkout the current patch)

Reply:
Can we just make it a static inline instead? It can live in the same header. I want to come up with a better name, too, but haven't been able to crack that yet...

Maybe something like `MPIR_Request_anysrc_disabled`?